nemo/collections/common/callbacks/ema.py

```diff
@@ -119,7 +119,7 @@ def on_load_checkpoint(
 
         # use the connector as NeMo calls the connector directly in the exp_manager when restoring.
         connector = trainer._checkpoint_connector
-        ckpt_path = connector.resume_checkpoint_path
+        ckpt_path = connector._ckpt_path
```

Comment:
This would be equivalent to replace these lines:
```py
ckpt_path = trainer.ckpt_path
```

and avoids calling protected API

Reply:
@awaelchli this has been fixed in the latest commit, thanks!